File: benchmark/results/ts-script-sonnet/run-002/b-baseline/output/src/domain/catalog/services/ProductService.ts
import { Product } from '../aggregates/Product';

/** Service providing business operations for catalog products. */
export class ProductService {
  create(input: Partial<Product>): Product {
    void input;
    throw new Error('not implemented');
  }

  findById(id: string): Product | null {
    void id;
    throw new Error('not implemented');
  }

  list(limit: number): Product[] {
    void limit;
    throw new Error('not implemented');
  }

  delete(id: string): void {
    void id;
    throw new Error('not implemented');
  }
}

File: benchmark/results/python-invocation-script/run-002/a-mcp/output/catalog/services/product_service.py
from typing import Optional
from catalog.aggregates.product import Product

"""ProductService service."""
class ProductService:
    def create(self, input: Product) -> Product:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Product]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> list[Product]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

